bool initialized_
std_msgs/Header data